app/routes.py

```diff
@@ -54,8 +54,8 @@ def extract_text(url: HttpUrl = Query(..., description="The URL to extract text
 class DownloadImageRequest(BaseModel):
     image_url: HttpUrl
     text: str
-    x: int
-    y: int
+    x: int = Field(..., description="X coordinate of the text position", example=10)
```

Comment:
Why are you making these ints and not floats?

Reply:
Accept floats and the transform them to ints

Reply:
Got it. I'll update the code to accept floats for the `x` and `y` coordinates and then transform them to integers within the FastAPI endpoint. This will ensure flexibility in the input while maintaining compatibility with the image drawing library. I'll make the necessary changes and update the PR shortly.